src/components/password.components.js

```diff
@@ -0,0 +1,98 @@
+import $ from 'jquery';
+
+export default class PasswordInput {
+    constructor(queryString, valid = false, required = true, disabled = false) {
+        this.div = $(queryString);
+        this.input = $(`${queryString} input[type="password"]`);
+        this.icon = $(`${queryString} input[type="password"]+i`);
+
+        this.valid = valid;
+        this.disabled = disabled;
+        this.required = required;
+
+        this.hasQuery(this.div, this.input, this.icon);
+        this.initFor();
+    }
+
+    hasQuery(div, input, icon) {
```

Comment:
These names do not reflect what the functions are doing
![image](https://user-images.githubusercontent.com/92385147/162191812-f477530f-b6fd-4271-9399-c82ee6102aea.png)